microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -195,4 +195,20 @@ Issued: 2023-06-23
 
 {product-title} release 4.13.4, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:3620[RHBA-2023:3620] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3614[RHSA-2023:3614] advisory.
 
+For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
+
+[id="microshift-4-13-5-dp"]
+=== RHBA-2023:3620 - {product-title} 4.13.5 bug fix and security update
+
+Issued: 2023-07-20
+
+{product-title} release 4.13.5, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4094[RHBA-2023:4094] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:4091[RHSA-2023:4091] advisory.
+
+For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
```

Comment:
This is repeated from line 198

Reply:
Yes, it's in every release.

Reply:
Ah, it threw me off that it was added twice in this PR.